src/pathfora.js

```diff
@@ -1437,24 +1412,25 @@
 
           // The top recommendation should be default if we couldn't
           // get one from the api
-          var rec = config.content[0]
+          var rec = config.content[0],
+              recImage = document.createElement('div'),
+              recMeta = document.createElement('div'),
+              recTitle = document.createElement('h4'),
+              recDesc = document.createElement('p');
+
           widgetContentUnit.href = rec.url;
 
-          var recImage = document.createElement('div');
```

Comment:
having the var definitions together at the top of the function makes for a difficult reading experience. probably worth adding comments where the definitions used to be to make it more readable.